airflow/www/package.json

```diff
@@ -57,16 +57,17 @@
   "dependencies": {
     "bootstrap-3-typeahead": "^4.0.2",
     "bootstrap-toggle": "^2.2.2",
+    "ci": "^1.0.0",
     "d3": "^3.4.4",
     "d3-tip": "^0.9.1",
-    "dagre-d3": "^0.4.18",
+    "dagre-d3": "^0.6.3",
```

Comment:
@potiuk did you verify if this is working fine on the frontend?
https://github.com/apache/airflow/pull/4713
https://github.com/apache/airflow/pull/3634